app/src/main/java/com/android/calendar/EventInfoFragment.java

```diff
@@ -488,6 +540,9 @@ public EventInfoFragment(Context context, Uri uri, long startMillis, long endMil
 
         setStyle(DialogFragment.STYLE_NO_TITLE, 0);
         mUri = uri;
+        if (mUri.getPathSegments().contains("tasks")) {
```

Comment:
are you sure this is distinctive?

Reply:
yes

Reply:
I'm pretty sure there are valid URIs with "tasks" in them that are not tasks. Can we find a better way?

Reply:
yes I try

Reply:
fixed